### PR TITLE
fix: windows

### PR DESCRIPTION
**Pull request by ryanthemanuel**
### User facing changelog

N/A

### Additional details

The main goal here is to fix flaky tests in windows. This was accomplished via:

* Bump the resource class of the windows executor
* Add some waits to a few places that seem to rely on `postFragment` subscriptions

Neither of these are ideal solutions; however, they don't appear to be new problems (I can recreate them in at least v10.0.3). Consequently we want to make the builds more stable and follow up on these issues later.

### Steps to test

Check that CI succeeds

### How has the user experience changed?

N/A

### PR Tasks

- [X] Have tests been added/updated?
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: circle.yml [hunk 1]
```diff
@@ -2619,15 +2619,15 @@ windows-workflow: &windows-workflow
...
+        resource_class: windows.xlarge
         requires:
           - windows-build
 
     - create-build-artifacts:
         <<: *full-windows-workflow-filters
         name: windows-create-build-artifacts
         executor: windows
-        resource_class: windows.medium
+        resource_class: windows.xlarge
```
Comment (lmiller1990) [inline circle.yml:2630]: A little unsettling that we don't know why this is required, but glad to have windows CI back to green.

File: packages/data-context/src/actions/MigrationActions.ts [hunk 2]
```diff
@@ -32,9 +32,6 @@ import {
 } from '../sources/migration'
```
Comment (lmiller1990) [inline packages/data-context/src/actions/MigrationActions.ts:32]: Just to clarify for reviewers: every file change below this line is reverted from https://github.com/cypress-io/cypress/pull/22118 and you don't need to review it to review this PR.

**Review by lmiller1990 — APPROVED** [review]
Looks good, I will find out why https://github.com/cypress-io/cypress/pull/22118 has (frustrating) windows on CI fails. I'm fairly sure they are just flake, they've passed before, but I'll find out.

**Comment by lmiller1990** [comment]
Hey @ryanthemanuel I do not think we need to revert #22118 anymore - can you just merge [this commit](https://github.com/cypress-io/cypress/commit/c5f1025e7a10b53372aaa6ef88ace74f2eed65c0) and it'll be fine. I checked out #22118, added that one commit, and CI is green. [Link to CI run with 100% windows tests passing first run](https://app.circleci.com/pipelines/github/cypress-io/cypress?branch=lmiller1990%2F21939-with-windows-fix&filter=all).

File: circle.yml (hunk 1 above)
Comment (mjhenkes) [reply circle.yml:2630]: Agreed. 😕

**Review by mjhenkes — APPROVED** [review]

**Pull request merged**